WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -97,3 +82,79 @@ extension SiteAddressViewController: NUXKeyboardResponder {
         keyboardWillHide(notification)
     }
 }
+
+
+
+private extension SiteAddressViewController {
+    // MARK: - Private methods
+
+    /// Localize the "Continue" button
+    ///
+    func localizePrimaryButton() {
+        let primaryTitle = displayStrings.continueButtonTitle
+        submitButton?.setTitle(primaryTitle, for: .normal)
+        submitButton?.setTitle(primaryTitle, for: .highlighted)
+    }
+
+    /// Registers all of the available TableViewCells
+    ///
+    func registerTableViewCells() {
+        let cells = [
+            InstructionTableViewCell.reuseIdentifier: InstructionTableViewCell.loadNib(),
+            TextFieldTableViewCell.reuseIdentifier: TextFieldTableViewCell.loadNib()
+        ]
+
+        for (reuseIdentifier, nib) in cells {
+            tableView.register(nib, forCellReuseIdentifier: reuseIdentifier)
+        }
+    }
+
+    /// Describes how the tableView rows should be rendered.
+    ///
+    func loadRows() {
+        rows = [.instructions, .siteAddress]
+    }
+
+    /// Configure cells
+    ///
+    func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
+        switch cell {
+        case let cell as InstructionTableViewCell:
+            configureInstruction(cell)
+        case let cell as TextFieldTableViewCell:
+            configureTextField(cell)
+        default:
+            DDLogError("Error: Unidentified tableViewCell type found.")
```

Comment:
I can't return UITableViewCell() here because the methods don't ask for one, but the default case can't be empty, so I settled on logging an error message.